mkosi

```diff
@@ -3135,14 +3107,16 @@ def load_args() -> CommandLineArguments:
     arg_debug = args.debug
 
     load_defaults(args)
-    find_nspawn_settings(args)
+
+    args_find_path(args, 'nspawn_settings', "mkosi.nspawn")
+    args_find_path(args, 'build_script',    "mkosi.build")
+    args_find_path(args, 'build_sources',   ".")
+    args_find_path(args, 'build_dir',       "mkosi.builddir/")
+    args_find_path(args, 'postinst_script', "mkosi.postinst");
+    args_find_path(args, 'output_dir',      "mkosi.output/");
```

Comment:
not that it matters, but is there are a reason for using `''` quotes on the second parameter, but `""` quotes on the third?

Reply:
It's written in the commit message. For visual distinction between variable names and paths.